1-exercises/D-do-while/exercise.js

```diff
@@ -7,9 +7,15 @@
 */
 
 function evenNumbersSum(n) {
-    // TODO
+  let result = "";
+  let i = 0;
+  do {
+    i += 1;
+    result += `${n * (n - 1)} `;
```

Comment:
This does output the correct result, but the idea is to use the loop to add up the values. This line is actually calculating the correct answer each time the loop executes, so the loop isn't needed. 
Try to refactor it so that on each pass of the loop, it adds to the running total.

Reply:
Sure, Thanks.
I will give it a try. :)